db/migrate/20240828191157_create_tests.rb

```diff
@@ -0,0 +1,12 @@
+class CreateTests < ActiveRecord::Migration[6.1]
+  def change
+    create_table :tests do |t|
+      t.string :title
+      t.integer :level, default: 0
+      t.references :user, null: false, foreign_key: true
```

Comment:
```suggestion
      t.references :author, null: false, foreign_key: { to_table: :users }
```
давай сделаем вот так, потом нам это будет полезно